ferveo/src/pvss.rs

```diff
@@ -132,7 +153,7 @@ impl<E: Pairing, T> PubliclyVerifiableSS<E, T> {
                 dkg.validators.len() as u32,
             ));
         }
-        // phi.zeroize(); // TODO zeroize?
+        phi.zeroize();
```

Comment:
A safer approach is to implement (or derive if possible) `Zeroize` for things that *can* be zeroized, and then implement (or derive if possible) `ZeroizeOnDrop` for things that `should` be zeroized. Then you won't have to worry about explicit zeroization. Local variables you can put in something like https://docs.rs/secrets and let RAII handle the zeroization.

Reply:
Or https://docs.rs/secrecy/latest/secrecy/ if you don't want to depend on `libsodium`. Or copy the thing from https://github.com/nucypher/rust-umbral/blob/master/umbral-pre/src/secret_box.rs :)

Reply:
I started playing around with `secret_box.rs` today, I think it could be a good fit provided I can figure out how to derive `serde::{Serialize, Deserialize}` on it

Reply:
Usually you would need `SecretBox` to manage temporary variables that have to be zeroized (e.g. to wrap an ephemeral scalar). For something that needs to be serialized, you derive `ZeroizeOnDrop`.